src/dawn/Optimizer/Interval.h

```diff
@@ -162,6 +168,22 @@ class Interval {
   static std::vector<Interval> computeGapIntervals(const Interval& axis,
                                                    const std::vector<Interval>& intervals);
 
+  /**
```

Comment:
alright, can we discuss this? The problem is that when I run the doxygen, the blocks below under 
```
@code 
...
@endcode
```
ends up in a single line if using `///`, and gets properly formatted with /**. 
Have you tried it yourself?

Reply:
so using 

```
/// @code
/// stencil_function avg {
///   storage in;
///   direction dir;
///
///   Do { return in(dir+1); }
/// };
/// @endcode
```
produces 
![image](https://user-images.githubusercontent.com/5988437/34247807-1e2ff266-e634-11e7-9193-ffe1e7008c8d.png)
is this not desired? I don't understand the issue

Reply:
For the doc like the one here
https://meteoswiss-apn.github.io/dawn/doxygen/group__support.html#gabd7c4384be4d3d00103eeb03bebee1e5

my doxgen produces the following 
![image](https://user-images.githubusercontent.com/6085761/34251644-1a1c770a-e641-11e7-9d84-a617fd9d5d03.png)

I dont know why, but using the /** works... 
but ok if it works for you I ll change it. But I ll need to find a solution cause with my system I can not generate decent doc

Reply:
However I dont think we really have a convention

https://github.com/MeteoSwiss-APN/dawn/blob/master/src/dawn-c/Types.h

so we should discuss what to we use and reformat everything to adhere to one convention. But let do that later. 
And I would like to understand also why my doc generation does not work with ///. Because of that I have a tendency to prefer /*

Reply:
Maybe it is your browser, if I look at 
https://meteoswiss-apn.github.io/dawn/doxygen/group__support.html#gabd7c4384be4d3d00103eeb03bebee1e5 
I have multiple lines of code...
Very odd indeed!
Sure, this is not a pressing issue, it just came up